packages/nimble-components/src/wafer-map/modules/data-manager.ts

```diff
@@ -49,34 +49,25 @@ export class DataManager {
 
     private readonly computations: Computations;
     private readonly prerendering: Prerendering;
-    private readonly dataMap: Map<string, WaferMapDie>;
 
     public constructor(wafermap: WaferMap) {
         this.computations = new Computations(wafermap.dies, wafermap.quadrant, {
-            width: wafermap.canvasSideLength,
-            height: wafermap.canvasSideLength
+            width: wafermap.canvas.width,
+            height: wafermap.canvas.height
         });
 
         this.prerendering = new Prerendering(
```

Comment:
Same as for the Computations constructor - since we are already changing parameters here, it might make more sense to pass in the wafermap reference, and deconstruct individual properties within the `Prerendering `object itself.